doc/source/train/user-guides/experiment-tracking.rst

```diff
@@ -1,195 +1,354 @@
+can work for distributed training with Ray Train. We will conclude the session with debugging
+tips.
 
-.. _train-experiment-tracking-native:
+Before we begin, the following is roughly how you can use the native experiment tracking library calls 
+inside of Ray Train. 
 
-Using native experiment tracking libraries
-------------------------------------------
+.. code-block:: python
 
-.. tab-set::
+    from ray.train.torch import TorchTrainer
+    from ray.train import ScalingConfig
 
-    .. tab-item:: PyTorch
+    def train_func(config):
+        # Training code and native lexperiment tracking library calls go here.
 
-        You can use experiment tracking libraries such as Weights & Biases, Mlflow, or
-        Comet directly in your Ray Train training loop.
+    scaling_config = ScalingConfig(num_workers=2, use_gpu=True)
+    trainer = TorchTrainer(train_func, scaling_config=scaling_config)
+    result = trainer.fit()
 
-        There are two things to keep in mind:
+Ray Train lets you use native experiment tracking libraries by customizing the tracking 
+logic inside the ``train_func`` function. In this way, you can port your experiment tracking 
+logic to Ray Train with minimal changes. 
 
-        1. Your code is executed in parallel on many workers. However, you often only want to report
-        results from one of these workers (usually the first worker - the "rank 0" worker).
+Log distributed training experiments
+====================================
 
-        2. When using the native libraries, you should report the results to both Ray Train and
-        the experiment tracking library.
+In this section, we break the interaction with experiment tracking backend into 4 logical steps:
+- Set up to connect to a tracking backend
+- Configure and launch a run
+- Log
+- Finish the run
 
-        Example:
+Let's dive into each one of them.
 
-        .. code-block:: python
+.. note::
+
+    A major difference between distributed and non-distributed training is that in distributed training, 
+    multiple processes are running in parallel and under certain setups they have the same results. If all 
+    of them are reported to the tracking backend, there will be duplicated results. To address that,  
+    Ray Train lets you apply logging logic to only the rank 0 worker with the following method:
+    :meth:`context.get_world_rank() <ray.train.context.TrainContext.get_world_rank>`.
+
+    .. code-block:: python
+
+        from ray import train
+        def train_func(config):
+            ...
+            if train.get_context().get_world_rank() == 0:
+                # do your logging logic only for rank0 worker.
+            ...
+
+
+Step 1: Set up necessary components to be able to connect to the tracking backend of your choice
+------------------------------------------------------------------------------------------------
+
+First, you should choose which tracking backend to use: W&B, MLflow, TensorBoard etc.
```

Comment:
For this section would it be helpful to showcase all the tracking services you use in this document? Specifically, Comet is referenced later but is the only one not mentioned in this introduction section.